Tasks/code-push-vsts-task/task.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "b69d19a0-7f6d-11e5-a86a-c199aa7a6cf0",
-    "name": "CodePush",
-    "friendlyName": "CodePush",
+    "name": "codepush-release",
+    "friendlyName": "CodePush - Release",
     "description": "A VSTS build task for releasing app updates to the CodePush service",
```

Comment:
Can we change "build" to "build/release"? Now that we're focusing on release management as well, we shouldn't tie the task to just build.